src/pages/Team/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+import { TEAM_LIST } from "./TeamList";
+
+import ImageCard from "@/components/ImageCard";
+
+import ConunLogoWhite from "@/assets/icons/conun_logo_white.svg";
+import styles from "./Team.module.scss";
+
+function Team() {
+  const [selected, setSelected] = useState("All");
+  const teamTabs = [
+    "All",
+    "Lead",
+    "Management / Planning",
+    "Development",
+    "Design",
+    "Marketing",
+    "Associate",
+  ];
+
+  return (
+    <div className={styles.TeamPage}>
+      <div className={styles.TeamIntroContainer}>
+        <ConunLogoWhite className={styles.ConunLogo} />
+
+        <div className={styles.ConunTitleContainer}>
+          <h1 className={styles.ConunTitle}>CONUN GROUP</h1>
+        </div>
+      </div>
+      <div className={styles.TeamContainer}>
+        <h3>Meet the team</h3>
+        <p>
+          CONUN is based in Seoul, South Korea. Since 2017 CONUN has strived to
```

Comment:
CONUN is based in Seoul, South Korea. Since 2017 CONUN has strived to
          be innovative and unique, working on projects that we hope will change
          the future. Our team members are diverse, coming from various locations
          across the globe...